.github/dependabot.yml

```diff
@@ -27,6 +27,10 @@ updates:
         # Guice 7.x only works with Jakarta EE and not Java EE
         versions:
           - ">= 6"
+      - dependency-name: "org.eclipse.persistence:org.eclipse.persistence.jpa"
+        # Version 4 is based on Jakarta EE
```

Comment:
So is version 3. For consistency with the other comments, can we make this say “Jakarta Persistence 3.x only works with Jakarta EE and not Java EE”?